l10n_it_fatturapa_out_triple_discount/data/invoice_it_template.xml

```diff
@@ -0,0 +1,43 @@
+<odoo>
+    <template
+        id="account_invoice_line_it_triple_discount"
+        inherit_id="l10n_it_fatturapa_out.account_invoice_line_it_sconto_maggiorazione"
+    >
+        <xpath expr="//ScontoMaggiorazione" position="after">
+            <t
+                t-set="importo2"
+                t-value="get_importo(line, discount_field='discount2')"
+            />
+            <t
+                t-set="importo3"
+                t-value="get_importo(line, discount_field='discount3')"
+            />
+            <ScontoMaggiorazione t-if="line.discount2 != 0 or importo2 != 0">
+            <!-- [2.2.1.10] -->
+            <t t-if="importo">
+                <Tipo t-if="importo2 &gt; 0">SC</Tipo>
+                <Tipo t-if="importo2 &lt;= 0">MG</Tipo>
+                <Importo t-esc="format_numbers(abs(importo2))" />
+            </t>
+            <t t-else="">
+                <Tipo t-if="line.discount2 &gt; 0">SC</Tipo>
+                <Tipo t-if="line.discount2 &lt;= 0">MG</Tipo>
+                <Percentuale t-esc="format_numbers_two(abs(line.discount2))" />
```

Comment:
Il modulo nella `12.0` aggiungeva solo il nodo `Percentuale`, è possibile mantenere il vecchio comportamento nella migrazione?
La modifica al comportamento esistente andrebbe almeno in un commit diverso, o meglio ancora in una PR diversa.

Anche perché lo sconto triplo potrebbe portare un importo scontato diverso a seconda di come vengono applicati gli sconti, ad esempio additivo o moltiplicativo o chissà cos'altro ci si può inventare con gli override di https://github.com/OCA/account-invoicing/blob/bab944e76aa71c3f300695fc90a4656a9055da4e/account_invoice_triple_discount/models/account_move_line.py#L97